includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -2087,7 +2087,7 @@ public static function print_status_meta_box( $post ) {
 							</strong>
 							<?php
 							esc_html_e(
-								'Review each instance of invalid markup and determine whether the action of removing (or keeping) is approved. You can change the status and click preview to see what impact the invalid markup has on the page. Invalid markup that is not approved is markup which you may not have seen yet or which you want to fix directly in underlying code.',
+								'For each detected invalid markup, determine whether it can be safely removed or it needs to be kept. You can change the Markup Status (Removed or Kept) and click the Preview Changes button to see its impact on the page. Once the proper action has been taken (Remove or Keep), you can toggle the checkbox the instance as Reviewed.',
```

Comment:
“markup” is a mass noun not a count noun (e.g. one doesn't hear “there are 4 markups”), so “for each detected invalid markup” doesn't quite work. So this is why instance is being used.

```suggestion
								'For each instance of invalid markup, determine whether it can be safely removed or it needs to be kept. You can change the Markup Status (Removed or Kept) and click the Preview Changes button to see its impact on the page. Once the proper action has been taken (Remove or Keep), you can toggle the checkbox the instance as Reviewed.',
```